uiowa/uiowa
# set max image size for uploads going forward

Resolves https://github.com/uiowa/uiowa/issues/4863

To Test

- `ddev composer install`
- Sync or install a site and see if you can upload an image larger than 2952x2952px. You will see a status message that the image has been resized to be under the max
- Check /media/add/image, wysiwyg upload, card image upload

Note: I will publish https://sitenow.uiowa.edu/node/326/latest once this is merged/deployed
